### ISSUE TITLE
Refactor proposal list to use CSS layouts

### ISSUE THREAD

**Issue by mprasanjith** [DXgovernance/dxvote#298]
The HTML table in the proposals list was troublesome while implementing some nice-to-have UI improvements. (https://github.com/DXgovernance/dxvote/pull/291) Down the line, we can look into refactoring the table component to a CSS version.

**Comment by luchux** [comment]
yeah, I don't like either the table markup. 
I migrated to this from something worst. We need to stop adding markup that styles widths of header and body separated (thats the oldest approach). I moved things to use CSS Grid that is a great way of avoid flexbox problems with dynamic with allocation for equal width columns. 
But I added another think i dislike too, I wrote things to use styled components styling `<table>, <tr>, <th>, <tbody> ` thinking semantic html5 was something important (till I saw the rest of the code and realized I went way too far).  Now we have too many constraint on the way we build the Tables. 

IMHO and in my experience the best has been to do a Table component that render divs, using flexbox or a combo grid-css and flexbox. 

Definitely keen to discuss this and work together if you want to define a nice reusable components, for `<Table></Table>` that let us have a header and a body with no hard styling, and that allow us to reuse everywhere in where atm there is or my approach or the old approach.
